search/includes/classes/class-health.php

```diff
@@ -131,7 +135,7 @@ public function validate_index_entity_count( array $query_args, \ElasticPress\In
 	 *      'post_status' => array( $post_statuses )
 	 * ];
 	 *
-	 * @param mixed $indexable Instance of an ElasticPress Indexable Object to search on
+	 * @param Comment|Post|Term|User $indexable Instance of an ElasticPress Indexable Object to search on
```

Comment:
Should this be `Indexable` instead?

Reply:
I am afraid it shouldn't — `Indexable` does not have a `format_args()` method used by the code. That's why I had to enumerate all classes that have.

Reply:
https://github.com/Automattic/vip-go-mu-plugins/blob/2a822a6f5202daaf9085d7d45acdac0d69ae2fd3/search/includes/classes/class-health.php#L145

Reply:
Hmm I see. Probably it should though? Maybe it should be an abstract class defining `format_args` as an abstract method. That it is how it is being used anyways I feel like, what do you think?

Reply:
I will address this in another PR: I saw several places where `Indexable` does not fit while working with the Search plugin; chances are that `format_args()` is not the only method.

Reply:
Actually, I am not sure this is even possible — those classes are defined in a third-party module, we cannot even attach an interface to them :-( The upstream will need to fix this.